browser/base/content/browser-sync.js

```diff
@@ -735,7 +741,12 @@ var gSync = {
   },
 
   async openFxAEmailFirstPage(entryPoint) {
-    const url = await FxAccounts.config.promiseConnectAccountURI(entryPoint);
+    let url;
+    if (RUST_BACKEND) {
+      url = await FxAccounts.config.promiseConnectAccountOAuthURI();
+    } else {
+      url = await FxAccounts.config.promiseConnectAccountURI(entryPoint);
```

Comment:
instead of short circuiting `promiseConnectAccountURI` we check if rust backend is enabled and start an oauth flow instead. This is a bit problematic because then this has to be updated in several places.